src/commonMain/kotlin/exchange.dydx.abacus/utils/AnalyticsUtils.kt

```diff
@@ -1,26 +1,97 @@
 package exchange.dydx.abacus.utils
 
 import exchange.dydx.abacus.output.SubaccountOrder
+import exchange.dydx.abacus.output.input.OrderType
 import exchange.dydx.abacus.state.manager.HumanReadableCancelOrderPayload
 import exchange.dydx.abacus.state.manager.HumanReadablePlaceOrderPayload
+import exchange.dydx.abacus.state.manager.HumanReadableTriggerOrdersPayload
+import kollections.JsExport
 import kollections.toIMap
+import kotlinx.serialization.Serializable
+
+@JsExport
+@Serializable
+enum class TriggerOrderAction(val rawValue: String) {
+    REPLACE("REPLACE"),
+    CANCEL("CANCEL"),
+    CREATE("CREATE"),
+    ;
+
+    companion object {
+        operator fun invoke(rawValue: String?) =
+            TriggerOrderAction.values().firstOrNull { it.rawValue == rawValue }
+    }
```

Comment:
We don't need this here - Kotlin has built-in string conversion for constant enums: https://kotlinlang.org/docs/enum-classes.html#working-with-enum-constants